packages/sapience/src/hooks/charts/useOrderBookData.ts

```diff
@@ -328,7 +360,7 @@ export function useOrderBookData({
     actualTickSpacing,
   ]);
 
-  // 6. Derive Order Book Levels from Processed Data
+  // 5. Derive Order Book Levels from Processed Data
```

Comment:
You're right this might make it weird with numeric/scalar markets. We can test and see in coming days